pkg/identity-manager/casbin_entity_adapter.go

```diff
@@ -0,0 +1,72 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package identitymanager
+
+import (
+	"fmt"
+
+	"github.com/casbin/casbin/model"
+	"github.com/casbin/casbin/persist"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/vmware/dispatch/pkg/entity-store"
+)
+
+type CasbinEntityAdapter struct {
+	store entitystore.EntityStore
+}
+
+func NewCasbinEntityAdapter(store entitystore.EntityStore) *CasbinEntityAdapter {
+	return &CasbinEntityAdapter{store: store}
+}
+
+func (a *CasbinEntityAdapter) LoadPolicy(model model.Model) error {
+	var policies []*Policy
+
+	// Policy status like CREATING, DELETING doesn't have any effect on
```

Comment:
Doesn't have any effect on what?

Reply:
I guess I context switched while typing that ;), I will fix it. Basically, I meant that LoadPolicy doesn't inspect the status of the policy entity. The status is more for the controllers to act upon.